### ISSUE TITLE
Autocomplete for user-defined procedures/functions?

### ISSUE THREAD

**Issue by ksowa** [dbeaver/dbeaver#9588]
**Is your feature request related to a problem? Please describe.**

I work a lot with MySQL databases, which use UUIDs stored as `BINARY(16)`. In order to convert them from and to a human-readable format, I've got 2 procedures defined in each of those databases. To get a record with a desired UUID, I have to run a query like:

```
SELECT *
FROM table_name
WHERE id = uuid_to_ouuid('664a3350-dc2b-11ea-9f33-0242ac130003')
```

where `uuid_to_ouuid` is my custom procedure. The problem is that I have to manually write its name every time, without any auto-completion. 

**Describe the solution you'd like**
It would be very usable to have auto complete of the custom procedures' names the same way as it completes built-in ones. 

**Additional context**
I'm attaching a few screenshots which show the thing I'm thinking about.

![image](https://user-images.githubusercontent.com/1027333/90642866-36462e80-e233-11ea-8fd7-3226483758bb.png)
![image](https://user-images.githubusercontent.com/1027333/90642237-6d681000-e232-11ea-8e01-2c6c84bc1fb9.png)
![image](https://user-images.githubusercontent.com/1027333/90642738-1151bb80-e233-11ea-9bc9-025322852868.png)

**Comment by elring** [comment]
Regarging the label x:mysql. This is not limited do Mysql. It would by nice to have function/procedure autocompletion also in PostgreSQL :-)

**Comment by serge-rider** [comment]
I think this feature is already there.
You can enable it in preferences:
![image](https://user-images.githubusercontent.com/6398845/95650929-6e5d4580-0aef-11eb-9cbf-cb9655fc5c13.png)

It is disabled by default because usually there are a lot of stored procedures and having this huge list in proposals is not very convenient.

**Comment by elring** [comment]
I see it now. Thanks. 
It has it's own problems (works only when global search is on and works only after FROM keyword - not directly after SELECT keyword and duplicates function parameters), but that's for another issue.

**Comment by hartois** [comment]
![image](https://user-images.githubusercontent.com/11641656/114295884-ca118700-9ab0-11eb-85aa-447a5498856f.png)

But I see only table "ttt", and does not see function "testexception".
Do I need to turn on something else?
![image](https://user-images.githubusercontent.com/11641656/114295880-bebe5b80-9ab0-11eb-8e19-f6d666ad70ef.png)
